src/connectedGraph.js

```diff
@@ -0,0 +1,31 @@
+export default function connectedGraph( graph = {} ){
+  if( graph == null ){
+    return 'Cannot check connection parity of null'
+  }
+  if( graph.constructor === Array || graph.constructor !== Object ){
+    return 'Please use objects'
+  }
+
+  let nodes = Object.keys(graph)
+  let node = nodes[0]
+  const alreadyVisited = []
+
+  const recursion = vertex => {
+    if(!alreadyVisited.includes(vertex)){
+      let destinations = graph[vertex]
+      alreadyVisited.push( vertex )
+      for( let e of destinations ){
+        recursion(e)
+      }
+    }
+  }
+
+  recursion(node)
+
+  for(let node2 of nodes){
+    if(!alreadyVisited.includes(node2)){
+      return false
+    }
+  }
+  return true
+}
```

Comment:
Nice very understandable and readable. I haven't done this problem. But when I do we can compare. In the mean think. Maybe you can refactor the code in many different way's and compare speed if you haven't, could be fun.